moveit_planners/pilz_industrial_motion_planner/src/trajectory_generator_circ.cpp

```diff
@@ -58,11 +58,9 @@ TrajectoryGeneratorCIRC::TrajectoryGeneratorCIRC(const moveit::core::RobotModelC
                                                  const std::string& /*group_name*/)
   : TrajectoryGenerator::TrajectoryGenerator(robot_model, planner_limits)
 {
-  if (!planner_limits_.hasFullCartesianLimits())
-  {
-    throw TrajectoryGeneratorInvalidLimitsException(
-        "Cartesian limits are not fully set for CIRC trajectory generator.");
-  }
+  // TODO: Need to check if all the parameters are set up the user in cartesian_limits.yaml
```

Comment:
Can't we just check if the parameters have an override?

Reply:
The current way she's done it with no defaults will fail to load if the user does not specify all the required parameters.